jquery.dynatable.js

```diff
@@ -1254,6 +1255,10 @@
             text: settings.inputs.searchText
           }).append($search);
 
+      if (settings.inputs.searchAutofocus) {
+        $($search).attr('autofocus', 'autofocus');
```

Comment:
When using jQuery, we would generally use the `prop` function rather than the `attr` function for this. Also, the `$search` element is already a jQuery element, so it doesn't need to be re-wrapped. Probably instead use something like:

```js
$search.prop('autofocus', true)
```